NuGet.Config

```diff
@@ -2,6 +2,6 @@
 <configuration>
   <packageSources>
     <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v2" />
-    <add key="NuGet" value="https://nuget.org/api/v2/" />
+    <add key="NuGet" value="https://www.nuget.org/api/v2/" />
```

Comment:
Looks like there is a regression somewhere with redirect. You were commenting on the related issue.